test/native/helpers.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import { act, render, fireEvent } from '@testing-library/react-native';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	subscribeParentGetHtml,
+	provideToNative_Html as provideToNativeHtml,
+} from '@wordpress/react-native-bridge';
+// Editor component is not exposed in the pacakge because is meant to be consumed
+// internally, however we require it for rendering the editor in integration tests,
+// for this reason it's imported with path access.
+// eslint-disable-next-line no-restricted-syntax
+import Editor from '@wordpress/edit-post/src/editor';
+
+// Set up the mocks for getting the HTML output of the editor
+let triggerHtmlSerialization;
+let serializedHtml;
+subscribeParentGetHtml.mockImplementation( ( callback ) => {
+	if ( ! triggerHtmlSerialization ) {
+		triggerHtmlSerialization = callback;
+		return {
+			remove: () => {
+				triggerHtmlSerialization = undefined;
+			},
```

Comment:
I realized that I forgot that this part required to return a function to remove the listener.